test/propagatortest.jl

```diff
@@ -0,0 +1,33 @@
+using Base.Test
```

Comment:
This is redundant. But you will need `using QuBase` either here or in `runtests.jl`.